packages/web-workers/src/messenger/iframe/remote-origin.ts

```diff
@@ -0,0 +1,9 @@
+import {createIframeWorkerMessenger} from './create-messenger';
+
+export function createRemoteOriginIframeMessengerFactory(iframeSrc: string) {
```

Comment:
@lemonmade after I used 🎩  in an actual project, I noticed that the signature was off. 

the `createWorkerMessenger` strategy function is actually being called with the URL later by remote-ui itself, but we need to setup the iframeSrc earlier. So I had to  change it to be a factory function.